drools-wb-webapp/pom.xml

```diff
@@ -438,6 +438,10 @@
       <groupId>org.kie.workbench.services</groupId>
       <artifactId>kie-wb-common-datamodel-backend</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.drools</groupId>
+      <artifactId>drools-wb-drl-extension-server</artifactId>
+    </dependency>
```

Comment:
@manstis, just a question, looking on the related PR for kie-wb-distributions, isn't here need for the scope runtime?

Reply:
@jomarko AFAIK the ```runtime``` scope in ```kie-wb-distributions``` webapps was added by @cristianonicolai to shorten the GWT compiler classpath as much as possible. It isn't as essential for ```drools-wb``` (and in fact no other dependencies have ```runtime``` scope in this module).

Reply:
@manstis, thanks, then if you are fine, I am fine with the scope.

Reply:
correct, the goal is to reduce the classpath for the GWT compiler. Although not necessary for showcase apps in general, I would still recommend doing it because it has the potential to at least speed things up.